### ISSUE TITLE
Fonts in italic after setting the first one as italic

### ISSUE THREAD

**Issue by dimitre** [plotdevice/plotdevice#35]
I have to explicitly declare a font as non-italic after setting the italic version

``` python
font('PMN Caecilia', 'heavy', 160, italic=True)
# Italic
font('PMN Caecilia', 'heavy', 160)
# Italic too
font('PMN Caecilia', 'heavy', 160, italic=False)
# non italic
```

**Comment by samizdatco** [comment]
I agree that's confusing. The idea was that any unspecified parameters would be carried over from the current settings, allowing for incremental style modifications like:

```python
font('Avenir Next', 64, 'heavy')
text('This is the Headline', 25, 100)
font(size=32, sc=True)
text('this is the subhead', 25, 150)
```

But it's definitely weird to see previous settings intruding when using it the way you've been—with positional args setting a new baseline style rather than keyword args modifying the existing one.

**Comment by samizdatco** [comment]
fixed in 210492db7190be63001438236416c41dadb6cb86
